rust/src/dns/log.rs

```diff
@@ -503,6 +503,7 @@ fn dns_log_json_answer(js: &mut JsonBuilder, response: &DNSResponse, flags: u64)
     if header.flags & 0x0080 != 0 {
         js.set_bool("ra", true)?;
     }
+    js.set_uint("z", ((header.flags & 0x0070) >> 4).into());
```

Comment:
Why int when bool is used for the other flags?

Reply:
@jasonish The answer is that used the same method that was used in Zeeks parser: https://github.com/zeek/zeek/blob/master/src/analyzer/protocol/dns/DNS.cc
They are also using bools for the other fields, but an int for this one.

Reply:
Yeah, I think bool makes more sense. Fits better with how we log the other flags.